docs/features/requirements.md

```
@@ -18,6 +18,60 @@ OVN should work with any supported OVS release, extra requirements for OVS versi
 
 ## UDN
+
+### Masquerade Subnet Sizing
+
+When UserDefinedNetworks are in use, the `internalMasqueradeSubnet` (IPv4) and
+`internalMasqueradeV6Subnet` (IPv6) must be large enough to accommodate the maximum
+number of concurrent networks (`MaxNetworks = 4096`).
+
+The masquerade IP index formula (from `go-controller/pkg/generator/udn/masquerade_ips.go`):
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)